components/uniset/src/uniset.rs

```diff
@@ -666,7 +691,7 @@ mod tests {
     fn test_serde_deserialize() {
         let inv_list_str = "[65,70,75,85]";
         let exp_inv_list = vec![0x41, 0x46, 0x4B, 0x55];
-        let exp_uniset = UnicodeSet::from_inversion_list(exp_inv_list).unwrap();
+        let exp_uniset = UnicodeSet::clone_from_inversion_list(exp_inv_list).unwrap();
         let act_uniset: UnicodeSet = serde_json::from_str(inv_list_str).unwrap();
         assert_eq!(act_uniset, exp_uniset);
     }
```

Comment:
Nit: Please add a test deserializing UnicodeSet from Postcard and assert that you get the UnicodeSet in borrowed mode (check the variant of the ZeroVec).